cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/workers/StatisticsWorker.java

```diff
@@ -104,28 +103,37 @@ public void stop() {
 
         // ensure that cache is empty before executor service initiating shutdown
         if (bookCounterCachesNotEmpty()) {
-            logger.info("Waiting statistics cache depletion");
+            LOGGER.info("Waiting statistics cache depletion");
             while (bookCounterCachesNotEmpty())
                 try {
                     Thread.sleep(100); // FIXME: find another way to wait the empty cache state
                 } catch (InterruptedException e) {
-                    logger.error("Interrupted while waiting statistics cache depletion");
+                    LOGGER.error("Interrupted while waiting statistics cache depletion");
                 }
         }
 
         // shut down executor service and wait for current job to complete
-        logger.info("Shutting down StatisticsWorker executor");
+        LOGGER.info("Shutting down StatisticsWorker executor");
         executorService.shutdown();
         try {
-            logger.debug("Waiting StatisticsWorker jobs to complete");
-            executorService.awaitTermination(Long.MAX_VALUE, TimeUnit.SECONDS); // FIXME: limit await timeout
-            logger.debug("StatisticsWorker shutdown complete");
+            LOGGER.debug("Waiting StatisticsWorker jobs to complete");
+            if (executorService.awaitTermination(Long.MAX_VALUE, TimeUnit.SECONDS)) {
```

Comment:
note: since we use `Long.MAX_VALUE`, I don't believe we will ever visit the `else` branch